commit 1714fc06e08db35fc3638ddde259d61c7a764861
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Thu Jan 4 00:54:13 2018 +0100

    hurd: Fix support/support_enter_mount_namespace.c build
    
            * support/support_enter_mount_namespace.c [!CLONE_NEWNS]: Do not
            include <sys/mount.h>.

---
 support/support_enter_mount_namespace.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/support/support_enter_mount_namespace.c
+++ b/support/support_enter_mount_namespace.c
@@ -20,7 +20,9 @@
 
 #include <sched.h>
 #include <stdio.h>
+#ifdef CLONE_NEWNS
 #include <sys/mount.h>
+#endif /* CLONE_NEWNS */
 
 bool
 support_enter_mount_namespace (void)
